people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Team
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
+- Wes McKinney
+- Wouter Overmeire
+- Jeff Reback
+- Skipper Seabold
+- Chang She
+- Sinhrks
+- Jeff Tratner
+- Joris Van den Bossche
+
+## Project NumFOCUS Board
```

Comment:
@wesm You included here this list, but in the main governance document you left out the section 'NumFOCUS Subcommittee' compared to the Jupyter version. 
I don't know if this board is meant to be similar as the Subcommittee in the Jupyter document, but now it is not clear what this board actually means.